timesketch/ui/graphs/graph-view.component.ts

```diff
@@ -31,12 +31,56 @@ export class GraphViewComponent {
 
   constructor(private readonly changeDetectorRef: ChangeDetectorRef) {}
 
+  showNeighborhood(event) {
+    let neighborhoodCollection = event.cy.collection()
+    const selectedElements = event.cy.filter(':selected')
+
+    if (!selectedElements.length) {
+      event.cy.elements().removeClass('faded')
+      return
+    }
+
+    selectedElements.forEach(function (element) {
```

Comment:
If you decide to leave the loop as it is, please use arrow function.

Reply:
Removed the loop.